dev/breeze/src/airflow_breeze/commands/kubernetes_commands.py

```diff
@@ -1574,11 +1435,12 @@ def _run_complete_tests(
         output=output,
         num_tries=num_tries,
         force_recreate_cluster=force_recreate_cluster,
-        verbose=verbose,
-        dry_run=dry_run,
     )
     if returncode != 0:
-        _logs(python=python, kubernetes_version=kubernetes_version, dry_run=dry_run, verbose=verbose)
+        _logs(
+            python=python,
+            kubernetes_version=kubernetes_version,
+        )
```

Comment:
Here and several times below, I'd drop the trailing comma so this collapses down.

```suggestion
        _logs(python=python, kubernetes_version=kubernetes_version)
```